### DISCUSSION TITLE
POST query failed while passing Int in body.

### DISCUSSION THREAD

**Discussion by yagnesh16697** [hyperjumptech/monika#923] [General]
![image](https://user-images.githubusercontent.com/40670389/196353968-30e112f9-a8f8-4b81-8082-20abf1c71c6c.png)
![image](https://user-images.githubusercontent.com/40670389/196354020-9cb025b0-6cbc-41c8-9a7d-36686220ef45.png)
![image](https://user-images.githubusercontent.com/40670389/196354120-60d17acd-49db-48f9-832f-ef0a5069f506.png)

So basically I am trying to POST request using POSTMAN and it is working as expected. While in YAML file it is giving me an error that OTP must be an Integer. So it converts it into other data type and returning an error.  

Tried with !!int 473682 but still giving me same error.

Req Body: 
![image](https://user-images.githubusercontent.com/40670389/196354748-fe448e78-9c29-423b-9fd0-84542c6df7bf.png)

**Comment by sapiderman** [comment]
@yagnesh16697 thanks for the feedback.
We fixed this bug, its been merged in #929. Let us know if you have any feedback!
